tests/integration/cli/test_accounts.py

```diff
@@ -67,6 +67,36 @@ def test_import_valid_private_key(ape_cli, runner, temp_account, temp_keyfile_pa
     assert temp_keyfile_path.is_file()
 
 
+@run_once
+def test_import_alias_is_private_key(ape_cli, runner):
+    # Attempt using private key as the alias.
+    key_alias = f"0x{PRIVATE_KEY}"
+    result = runner.invoke(
+        ape_cli,
+        ["accounts", "import", key_alias],
+        input="\n".join([f"0x{PRIVATE_KEY}", PASSWORD, PASSWORD]),
+    )
+    assert result.exit_code != 0, result.output
+    expected = "ERROR: (AccountsError) Longer aliases cannot be hex strings.\n"
+    assert result.output == expected
+
+
+@run_once
+def test_import_alias_is_really_long(ape_cli, runner):
+    """
+    For entropy related use-cases regarding alias, we
+    must ensure long aliases are supported.
+    """
+
+    long_alias = "this is a long alias that i am going to use and you cant stop me"
```

Comment:
@fubuloubu I added a test to make sure that we will always be able to support your sentence-long alias use-case